src/olympia/blocklist/models.py

```diff
@@ -361,9 +362,11 @@ def save(self, *args, **kwargs):
     @classmethod
     def _split_guids_full(cls, guids, v_min, v_max, filter_existing):
         # load all the Addon instances together
-        addon_qs = Addon.unfiltered.filter(guid__in=guids).order_by(
-            '-average_daily_users').only_translations()
-        addons = list(addon_qs)
+        addons_qs = Addon.unfiltered.filter(guid__in=guids).only_translations()
+        addons = sorted(
```

Comment:
How are these new lines different from what the old lines were doing?

Reply:
they're the same - the follow on patch does it this way and I brought it forward (or left it in, depending on how you think about it)